wire-ios-request-strategy/Sources/Payloads/Processing/PayloadProcessing+Conversation.swift

```diff
@@ -136,21 +154,27 @@ extension Payload.Conversation {
         updateConversationStatus(for: conversation)
 
         conversation.needsToBeUpdatedFromBackend = false
+
+        return conversation
     }
 
-    func updateOrCreateSelfConversation(in context: NSManagedObjectContext,
-                                        serverTimestamp: Date,
-                                        source: Source) {
+    func updateOrCreateSelfConversation(
```

Comment:
should this also have `@discardableResult`?

Reply:
yes, fixed